server/routes/items.js

```diff
@@ -44,6 +48,20 @@ router.post(
       isAuth,
       isIdentity,
     } = req.body;
+
+    // exchange the public token for a private access token and store with the item.
+    const response = await plaid.itemPublicTokenExchange({
```

Comment:
would be nice to have some error handling for these API calls, but you can always just add error handling in a later PR